src/components/views/TasksManager.tsx

```diff
@@ -160,18 +158,22 @@ const TasksManager: FC = () => {
   const handleDragStart = (event: DragStartEvent): void => {
     setActiveTask(tasks.find((ta) => ta.id === event.active.id) ?? null);
   };
+
+  const handleHide = (label: string, hide: boolean): void => {
+    const newHiddenLists = {
+      ...hiddenLists,
+      [label]: hide,
+    };
+    setHiddenLists(newHiddenLists);
```

Comment:
Although your implementation is not wrong, we might prefer a different approach:

You can provide an updater function to the SetState method, you get the previous state for free and this solves a lot of issues with out-of-date state 😉 
```suggestion
    setHiddenLists((prevState) => ({
      ...prevState,
      [label]: hide,
    }));
```